build.gradle

```diff
@@ -35,7 +35,8 @@ dependencies {
 	compile gradleApi()
 	compile 'biz.aQute.bnd:bndlib:2.2.0'
 //	compile 'org.osgi:org.osgi.core:4.3.0'
-	compile 'org.eclipse.tycho:org.eclipse.osgi:3.6.2.R36x_v20110210' // provides EclipseEnvironmentInfo
+	compile 'commons-io:commons-io:2.4'
+	compile 'org.eclipse.tycho:org.eclipse.jdt.core:3.11.1.v20150902-1521'
```

Comment:
Is really a JDT dependency needed? From the other changes I didn't see any reference to that.